src/runtime/methodbinder.cs

```diff
@@ -677,14 +723,21 @@ static Type TryComputeClrArgumentType(Type parameterType, IntPtr argument, bool
                         defaultArgList.Add(parameters[v].GetDefaultValue());
                         defaultsNeeded++;
                     }
-                    else if (!paramsArray)
+                    else if (!isOperator && !paramsArray)
                     {
+                        // this is separate above because an operator method cannot have
+                        // keyword args, default args, or params arrays (exclusive cases)
                         match = false;
                     }
                 }
+                if (isOperator && defaultArgList.Count == 0)
+                {
+                    // If no default arguments were provided for an operable object.
+                    defaultArgList = null;
+                }
```

Comment:
Operator methods can enter the branch this is in (`pynargs < clrnargs && (!paramsArray || pynargs == clrnargs - 1))`) because there will be 2 clr args (`a` and `b`) but only one py arg (`b`), since the method is bound to `a` (`a.__add__(b)`). So we don't want to set `match = false` since it's still considered a match.

Reply:
Thanks for the catch